nbs/styles.css

```diff
@@ -35,3 +35,7 @@ div.description {
   font-size: 135%;
```

Comment:
_Commented on notebook `nbs/styles.css` Cell 1 Line 40_
```css
  padding-top: 5px;
  font-style: italic;
  font-size: 135%;
  opacity: 70%;
}

img {
  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1), 0 1px 3px rgba(0, 0, 0, 0.08);
```
other comment

_Respond and view the context [here](http://localhost:3000/GitNotebooks/docs/pull/2)._